src/content/learn/editor-setup.md

```diff
@@ -1,62 +1,62 @@
 ---
-title: Editor Setup
+title: Configuration de l'éditeur
 ---
 
 <Intro>
 
-A properly configured editor can make code clearer to read and faster to write. It can even help you catch bugs as you write them! If this is your first time setting up an editor or you're looking to tune up your current editor, we have a few recommendations.
+Un éditeur correctement configuré rend le code plus clair à lire et plus rapide à écrire. Il peut aussi vous aider à corriger les bugs dès que vous les écrivez ! Si c'est la première fois que vous configurez un éditeur ou que vous cherchez à améliorer votre éditeur actuel, nous avons quelques recommandations. 
 
 </Intro>
 
 <YouWillLearn>
 
-* What the most popular editors are
-* How to format your code automatically
+* Quels sont les éditeurs les plus populaires
+* Comment formatter votre code automatiquement
 
 </YouWillLearn>
 
-## Your editor {/*your-editor*/}
+## Votre éditeur {/*your-editor*/}
 
-[VS Code](https://code.visualstudio.com/) is one of the most popular editors in use today. It has a large marketplace of extensions and integrates well with popular services like GitHub. Most of the features listed below can be added to VS Code as extensions as well, making it highly configurable!
+[VS Code](https://code.visualstudio.com/) est l'un des éditeurs les plus populaires aujourd'hui. Il a un grand magasin d'extensions et s'intègre bien avec des services populaires comme GitHub. La plupart des fonctionnalités listées ci-dessous peuvent être ajoutées à VS Code _via_ des extensions, ce qui le rend très configurable !
 
-Other popular text editors used in the React community include:
+Voici d'autres éditeurs utilisés dans la communauté React :
 
-* [WebStorm](https://www.jetbrains.com/webstorm/) is an integrated development environment designed specifically for JavaScript.
-* [Sublime Text](https://www.sublimetext.com/) has support for JSX and TypeScript, [syntax highlighting](https://stackoverflow.com/a/70960574/458193) and autocomplete built in.
-* [Vim](https://www.vim.org/) is a highly configurable text editor built to make creating and changing any kind of text very efficient. It is included as "vi" with most UNIX systems and with Apple OS X.
+* [WebStorm](https://www.jetbrains.com/webstorm/) est un environnement de développement intégré (_Integrated Development Environment ou IDE, NdT_) spécialement conçu pour JavaScript.
+* [Sublime Text](https://www.sublimetext.com/) supporte JSX et TypeScript, [la coloration syntaxique](https://stackoverflow.com/a/70960574/458193) et inclut l'autocomplétion.
```

Comment:
```suggestion
* [Sublime Text](https://www.sublimetext.com/) prend nativement en charge JSX et TypeScript, [la coloration syntaxique](https://stackoverflow.com/a/70960574/458193) et l'autocomplétion.
```

Le "built-in" s'applique aux trois lots de fonctionnalités. Attention au faux-ami hyper classique "to support" -> "supporter", qui est en fait "prendre en charge" (on "supporte" un truc pénible, on "prend en charge" une capacité)